locust/test/test_runners.py

```diff
@@ -906,7 +906,6 @@ def _(*args, **kwargs):
             # give time for users to generate stats, and stats to be sent to master
             sleep(0.1)
             master_env.events.quitting.fire(environment=master_env, reverse=True)
-            master.quit()
```

Comment:
Wouldnt it make more sense to remove the other line instead?

Reply:
I guess this test case is test quit event?

Reply:
Yes, but the quit event would normally not be triggered by the user directly.

Reply:
changed